modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsStore.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ignite.internal.processors.query.stat;
+
+import java.util.Collection;
+
+/**
+ * Statistics store interface.
+ */
+public interface IgniteStatisticsStore {
+    /**
+     * Clear statistics of any type for any objects;
+     */
+    public void clearAllStatistics();
+
+    /**
+     * Replace all tables partition statistics with specified ones.
+     *
+     * @param key Statistics key to replace statistics by.
+     * @param statistics Collection of partition level statistics.
+     */
+    public void replaceLocalPartitionsStatistics(StatsKey key, Collection<ObjectPartitionStatisticsImpl> statistics);
+
+    /**
+     * Get local partition statistics by specified object.
+     *
+     * @param key Key to get statistics by.
+     * @return Collection of partitions statistics.
+     */
+    public Collection<ObjectPartitionStatisticsImpl> getLocalPartitionsStatistics(StatsKey key);
```

Comment:
Why 'local'?
Does it mean we store statistics for local partitions only? primary or backups or both?
May it worth to rename interface?

Reply:
Because it should return only statistics for locals (both primary and backups) partitions.
Didn't get the idea, what name you suppose?